client/blocks/inline-help/inline-help-contact-page.tsx

```diff
@@ -1,17 +1,46 @@
 import { Button, Gridicon } from '@automattic/components';
+import { useSupportAvailability } from '@automattic/data-stores';
 import { Icon, comment } from '@wordpress/icons';
 import { useI18n } from '@wordpress/react-i18n';
 import classnames from 'classnames';
+import { useEffect, useState } from 'react';
 import InlineHelpSearchResults from './inline-help-search-results';
 import Mail from './mail-icon';
 
 interface Props {
 	closeContactPage: () => void;
 	onSelectResource: () => void;
+	setContactFormOpen: ( formMode: string ) => void;
 }
 
-const InlineHelpContactPage: React.FC< Props > = ( { closeContactPage, onSelectResource } ) => {
+const InlineHelpContactPage: React.FC< Props > = ( {
+	closeContactPage,
+	onSelectResource,
+	setContactFormOpen,
+} ) => {
 	const { __ } = useI18n();
+	const [ chatAvailable, setChatAvailable ] = useState( false );
+
+	const { data: dataChat, isLoading: isLoadingChat } = useSupportAvailability( 'CHAT' );
+	const { data: dataEmail, isLoading: isLoadingEmail } = useSupportAvailability( 'EMAIL' );
+
+	useEffect( () => {
```

Comment:
This is a bit unclear. Deserves a comment.

```suggestion
	// open the contact form automatically if the user has one option.
	useEffect( () => {
```